### ISSUE TITLE
feature request: Image Converter Tool

### ISSUE THREAD

**Issue by riginoommen** [drodil/backstage-plugin-toolbox#74]
As developers and designers image conversion tools with resizing support is more helpful 

1) SVG to PNG
2) PNG to SVG
3) PNG to JPG
4) JPG to PNG
5) SVG to JPG
6) JPG to SVG

**Comment by drodil** [comment]
Good idea but probably this requires to expand this with a backend plugin.

**Comment by deshmukhmayur** [comment]
Another use case would be to convert images to [data uris](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs).

Also, @drodil is it possible to process the images in browser (using something like [image-js](https://github.com/image-js/image-js)? I guess it can be too intensive depending on the image size. So a backend plugin would still be ideal for better performance.

**Comment by riginoommen** [comment]
Adding a backend plugin helps to manage the load also and make the plgun in more scalable and performant.

**Comment by drodil** [comment]
Hey there, now there is support to create tools that utilize the backend so this tool could be now done 👍 Any chance you would like to create a PR? Ping @riginoommen
